PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalEditPendingRequest.kt

```diff
@@ -0,0 +1,23 @@
+package com.braintreepayments.api.paypal
+
+/**
+ * A pending request for the PayPal web-based authentication flow created by invoking
+ * [PayPalLauncher.launch]. This pending request should be stored locally within the app or
+ * on-device and used to deliver a result of the browser flow in
+ * [PayPalLauncher.handleReturnToApp]
+ */
+sealed class PayPalEditPendingRequest {
```

Comment:
Let's rename to `PayPalVaultEditPendingRequest` and move to `vaultedit` directory